packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts

```diff
@@ -187,13 +183,13 @@ describe("Garbage Collection configurations", () => {
 			assert(!gc.configs.gcEnabled, "gcEnabled incorrect");
 			assert.equal(gc.configs.gcVersionInBaseSnapshot, 0, "gcVersionInBaseSnapshot incorrect");
 		});
-		it("gcFeature 0, Sweep enabled via gcGeneration", () => {
```

Comment:
Why remove this test case? I guess it's kind of a weird one, but supported?

Reply:
With this change, only `sweepEnabled` is exposed which means whether it's enabled for this session. If gcFeature is 0, GC and sweep are both disabled. That's why I changed what this tests does - didn't remove it.

Reply:
@agarwal-navin you can probably revert a lot of changes here now that you added back sweepAllowed.  I'd prefer we maintain coverage of that, but if you're out of time so be it

Reply:
Good point. I added sweepAllowed checks back for these.